c/makeotf/makeotf_lib/source/hotconv/sfnt.c

```diff
@@ -57,9 +54,6 @@ static Funcs g_funcs[] = {
     {name_, nameNew, nameFill, nameWrite, nameReuse, nameFree, 3, 5, 0},
     {cmap_, cmapNew, cmapFill, cmapWrite, cmapReuse, cmapFree, 1, 6, 0},
     {post_, postNew, postFill, postWrite, postReuse, postFree, 1, 7, 0},
-    {fvar_, fvarNew, fvarFill, fvarWrite, fvarReuse, fvarFree, 1, 8, 0},
-    {MMSD_, MMSDNew, MMSDFill, MMSDWrite, MMSDReuse, MMSDFree, 1, 9, 0},
-    {MMFX_, MMFXNew, MMFXFill, MMFXWrite, MMFXReuse, MMFXFree, 1, 10, 0},
     {CFF__, CFF_New, CFF_Fill, CFF_Write, CFF_Reuse, CFF_Free, 1, 11, 0},
```

Comment:
You need to change the second-to-last column to be consecutive numbers like this:
```c
static Funcs g_funcs[] = {
    {head_, headNew, headFill, headWrite, headReuse, headFree, 1, 1, 0},
    {hhea_, hheaNew, hheaFill, hheaWrite, hheaReuse, hheaFree, 2, 2, 0},
    {maxp_, maxpNew, maxpFill, maxpWrite, maxpReuse, maxpFree, 1, 3, 0},
    {OS_2_, OS_2New, OS_2Fill, OS_2Write, OS_2Reuse, OS_2Free, 1, 4, 0},
    {name_, nameNew, nameFill, nameWrite, nameReuse, nameFree, 3, 5, 0},
    {cmap_, cmapNew, cmapFill, cmapWrite, cmapReuse, cmapFree, 1, 6, 0},
    {post_, postNew, postFill, postWrite, postReuse, postFree, 1, 7, 0},
    {CFF__, CFF_New, CFF_Fill, CFF_Write, CFF_Reuse, CFF_Free, 1, 8, 0},
    {hmtx_, hmtxNew, hmtxFill, hmtxWrite, hmtxReuse, hmtxFree, 1, 9, 0},
    {vhea_, vheaNew, vheaFill, vheaWrite, vheaReuse, vheaFree, 2, 10, 0},
    {vmtx_, vmtxNew, vmtxFill, vmtxWrite, vmtxReuse, vmtxFree, 1, 11, 0},
    {GDEF_, GDEFNew, GDEFFill, GDEFWrite, GDEFReuse, GDEFFree, 1, 12, 0},
    {GSUB_, GSUBNew, GSUBFill, GSUBWrite, GSUBReuse, GSUBFree, 1, 13, 0},
    {GPOS_, GPOSNew, GPOSFill, GPOSWrite, GPOSReuse, GPOSFree, 1, 14, 0},
    {BASE_, BASENew, BASEFill, BASEWrite, BASEReuse, BASEFree, 1, 15, 0},
    {VORG_, VORGNew, VORGFill, VORGWrite, VORGReuse, VORGFree, 1, 16, 0},
    {STAT_, STATNew, STATFill, STATWrite, STATReuse, STATFree, 1, 17, 0},
};
```
In the example I caught in my debugger, it was calling the wrong free function because of this.

Reply:
I knew it must be that array!